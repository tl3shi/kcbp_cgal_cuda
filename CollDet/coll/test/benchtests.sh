# regression tests with program 'bench'

echo "DOP tree"
./bench -n 5 500 -d 1.5 0.501 -T -g to -x 4 -A do
./bench -n 5 500 -d 1.5 0.501 -T -g sh -x 7 -A do
./bench -n 5 500 -d 1.5 0.499 -T -g pl -x 4 -A do

echo ""
echo "Boxtree"
./bench -n 5 500 -d 1.5 0.501 -T -g to -x 4 -A bx
./bench -n 5 500 -d 1.5 0.501 -T -g sh -x 7 -A bx
./bench -n 5 500 -d 1.5 0.499 -T -g pl -x 4 -A bx

