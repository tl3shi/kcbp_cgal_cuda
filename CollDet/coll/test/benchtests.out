DOP tree
DopTree:init: Dop::NumOri = 24
# dist:  num. colls.:
1.500     198
1.250     344
1.000     432
0.751     500
0.501     500
# num collisions = 1974
# num faces = 50
DopTree:init: Dop::NumOri = 24
# dist:  num. colls.:
1.500       0
1.250       0
1.000       0
0.751     500
0.501     500
# num collisions = 1000
# num faces = 320
DopTree:init: Dop::NumOri = 24
# dist:  num. colls.:
1.500      36
1.250      50
1.000     129
0.749     348
0.499     500
# num collisions = 1063
# num faces = 32

Boxtree
DopTree:init: Dop::NumOri = 24
# dist:  num. colls.:
1.500     198
1.250     344
1.000     432
0.751     500
0.501     500
# num collisions = 1974
# num faces = 50
DopTree:init: Dop::NumOri = 24
# dist:  num. colls.:
1.500       0
1.250       0
1.000       0
0.751     500
0.501     500
# num collisions = 1000
# num faces = 320
DopTree:init: Dop::NumOri = 24
# dist:  num. colls.:
1.500      36
1.250      50
1.000     129
0.749     348
0.499     500
# num collisions = 1063
# num faces = 32
